CHANGELOG.md

```diff
@@ -1,6 +1,27 @@
 Change Log / Release Log for mallocMC
 ================================================================
 
+2.4.0crp
+--------
+**Date:** 2020-04-28
+
+This release removes the boost dependency and switched to C++11.
```

Comment:
Perhaps capitalize `Boost` and `switches` instead of `switched`.

Reply:
Do you want to add the removal of Boost dependency to one of the bullet points below?